src/Core/Core.csproj

```diff
@@ -39,10 +39,10 @@
     <PackageReference Include="Microsoft.CSharp" Version="4.5.0" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="3.0.0" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="3.0.0" />
-    <PackageReference Include="Microsoft.Quantum.Compiler" Version="0.24.201332" />
-    <PackageReference Include="Microsoft.Quantum.CSharpGeneration" Version="0.24.201332" />
-    <PackageReference Include="Microsoft.Quantum.Simulators" Version="0.24.201332" />
-    <PackageReference Include="Microsoft.Quantum.QSharp.Core" Version="0.24.201332" />
+    <PackageReference Include="Microsoft.Quantum.Compiler" Version="0.24.41605-pull" />
+    <PackageReference Include="Microsoft.Quantum.CSharpGeneration" Version="0.24.41605-pull" />
+    <PackageReference Include="Microsoft.Quantum.Simulators" Version="0.24.41605-pull" />
+    <PackageReference Include="Microsoft.Quantum.QSharp.Core" Version="0.24.41605-pull" />
```

Comment:
FYI: the latest release version is 0.24.210930.

Reply:
Updated to latest beta using this feature branch.